libbeat/common/cfgwarn/cfgwarn.go

```diff
@@ -35,7 +35,10 @@ func Beta(format string, v ...interface{}) {
 // Deprecate logs a deprecation message.
 // The version string contains the version when the future will be removed
 func Deprecate(version string, format string, v ...interface{}) {
-	postfix := fmt.Sprintf(" Will be removed in version: %s", version)
+	var postfix string
+	if version != "" {
+		postfix = fmt.Sprintf(" Will be removed in version: %s", version)
+	}
```

Comment:
I think we should add a comment to the function's godoc that the `version` argument is optional and can be empty.